src/components/product-list-component/product-description/product-description.jsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faIndianRupee } from "@fortawesome/free-solid-svg-icons";
+import { useParams } from "react-router-dom";
+import { AiOutlineHeart } from "react-icons/ai";
+import { AiOutlineShoppingCart } from "react-icons/ai";
+import "./product-description.css";
+import Rating from "./Rating";
+import Footer from '../../footer-component/Footer';
+export const ProductDescription = () => {
+  const [productdetails, setProductDetails] = useState([]);
+
+  const { productId } = useParams();
+
+  const getProductdetails = async () => {
+    const response = await fetch(
+      `https://fakestoreapi.com/products/${productId}`
+    );
+
+    setProductDetails(await response.json());
+  };
+
+  useEffect(() => {
+    getProductdetails();
+  }, []);
+
+  return (
+    <>
+    <main className="description-container">
+
+      {/* <!-- Left Column /  Image --> */}
+
+      <div className="left-column">
+        <img
+          src={productdetails.image}
+          alt="products"
+          className="img"
+          width="600px"
+          height="600px"
+        />
+      </div>
+
+      {/* <!-- Right Column --> */}
+      <div className="right-column">
+
+        {/* <!-- Product Description Top --> */}
+        <div className="product-label">
+          <div className="title">
+            <h2>{productdetails.title}</h2>
+          </div>
+
+          {/* <!-- Product Rating--> */}
+          <div className="product-rating">
+            <h6>
+            {/* productdetails?.rating?.rate */}
```

Comment:
Let's also not add these comments here as well.. any other dev can easily able to understand without the comment. Please remove all the comments in the same file